fastapi/dependencies/utils.py

```diff
@@ -329,8 +353,12 @@ def is_coroutine_callable(call: Callable) -> bool:
 ]:
     values: Dict[str, Any] = {}
     errors: List[ErrorWrapper] = []
-    response = response or Response(  # type: ignore
-        content=None, status_code=None, headers=None, media_type=None, background=None
+    response = response or Response(
+        content=None,
+        status_code=None,  # type: ignore
```

Comment:
Needed to put the `# type: ignore` here to get mypy to pass (sadly this forces the line to become longer, but has the benefit of *only* applying to that one argument.)